code/modules/modular_computers/file_system/programs/ntmessenger.dm

```diff
@@ -283,6 +283,10 @@
 
 	if (prob(1))
 		message += " Sent from my PDA"
+	// SKYRAT EDIT BEGIN - PDA messages show a visible message; again!
+	user.visible_message(span_notice("[user]'s PDA rings out with the soft sound of keypresses."))
+	user.balloon_alert_to_viewers("sent a PDA message")
```

Comment:
I feel like it should be one or the other, but not both.

Also the argument about it being basically like talking in a radio makes sense too, I feel like that's something that should have a range of like, one or two tiles tops.